EasyPost.Tests/Fixture.cs

```diff
@@ -393,6 +400,7 @@ public static class Fixture
             get
             {
                 return new Dictionary<string, object>
+                // TODO: Consolidate the date to a variable and reuse the same date variable in both fields below
```

Comment:
Remove this TODO (create a single data variable and reuse it in both datetime fields below.